src/main.cpp

```diff
@@ -167,6 +189,20 @@ static int parse_argv(int argc, char *argv[], struct options *opt)
             opt->conf_dir = optarg;
             break;
         }
+        case 'g': {
+            int lvl = log_level_from_str(optarg);
+            if (lvl == -EINVAL) {
+                log_error("Invalid argument for debug-log-level = %s", optarg);
+                help(stderr);
+                return -EINVAL;
+            }
+            log_set_max_level(lvl);
+            break;
+        }
+        case 'v': {
+            log_set_max_level(LOG_DEBUG);
+            break;
+        }
```

Comment:
This commit is adding -v and -g parameters to csd. It wasn't handled before this patch